authorization-common/src/main/resources/legacyRights.json

```diff
@@ -53,6 +53,7 @@
   "datahub/deleteDatahubInstance": "datahub/write",
   "datahub/setDatahubMaintenanceMode": "datahub/write",
   "datahub/rotateAutoTlsCertDatahub": "datahub/write",
+  "datahub/refreshRecipes": "datahub/write",
```

Comment:
should i wait for that PR or what? i guess depends on how this and that PR progresses

Reply:
I think this is still needed since legacy authz technically supported, so this is ok for now, I will resolve conflicts in my PR because I think my PR should wait for 2.49 branch cutting